python_bindings/test/generators/complexpy_generator.py

```diff
@@ -65,9 +72,13 @@ def generate(self):
 
         g.scalar_output[()] = g.float_arg + g.int_arg
 
-        intermediate.compute_at(g.tuple_output, y);
-        intermediate.specialize(g.vectorize).vectorize(x, g.natural_vector_size(hl.Float(32)));
+        intermediate.compute_at(g.tuple_output, y)
+        (
+            intermediate.specialize(g.vectorize).vectorize(
```

Comment:
Something odd seems to have happened here

Reply:
Fixed